src/app/resperson/get.controller.js

```diff
@@ -0,0 +1,38 @@
+const logger = require('../../common/utils/logger')(__filename);
+const CookieModel = require('../../common/models/Cookie.class');
+const resPersonApi = require('../../common/services/resPersonApi');
+
+module.exports = async (req, res) => {
+  logger.debug('In responsible person get controller');
+  const cookie = new CookieModel(req);
+  const errMSg = { message: 'Failed to get saved responsible persons' };
```

Comment:
`nit`: This variable `errMSg` is too easy to conflate with `errMsg` from the `req.session`.

Reply:
renamed to errMessage to avoid confusion